### ISSUE TITLE
[BUG] ESLint makes error about absolute import of Vite's PublicDir path

### ISSUE THREAD

**Issue by nerdchanii** [hello-chat/extension#8]
### Current Behavior

ESLint makes error about absolute import of Vite's PublicDir path.

### Steps To Reproduce
run under command
```
yarn eslint .
```
then, you can meet error Logs.
```
...paths..src/App.tsx
3:22  error  Unable to resolve path to module '/vite.svg'  import/no-unresolved

✖ 1 problem (1 error, 0 warnings)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Environment
OS: Mac OS 13.4
Node: 16.4.0
yarn: 1.22.19

### Anything else
Linked issue #7 

>⚠️
>Vite links the absolute path to publicDir.
>ex) /vite.svg
>
>Failed to resolve eslint error for resources imported to absolute path imported from public folder.
>
>I think it would be good to solve it through alias about public path with `@public` prefix.

I think it should be solved in a new issue after this PR is combined.

Create a new issue and link this PR.

_Originally posted by @nerdchanii in https://github.com/hello-chat/extension/issues/7#issuecomment-1622859613_

**Comment by nerdchanii** [comment]
I asked to `eslint-plugin-import`, they recommended(?) [`eslint-import-resolver-vite`](https://github.com/pzmosquito/eslint-import-resolver-vite)

**Comment by nerdchanii** [comment]
Oops. It's temporary resolved in #14 with ignore rules. 
cause, their commits pass lint-staged without ignore rules, git throw eslint-error.
